pyquil/gates.py

```diff
@@ -1150,11 +1102,8 @@ def FENCE(*qubits: Union[int, Qubit, FormalArgument]) -> Union[FenceAll, Fence]:
     "NOP": NOP,
     "HALT": HALT,
     "MEASURE": MEASURE,
-    "TRUE": TRUE,
```

Comment:
@notmgsk Should these be removed altogether or remapped? Remapping for `TRUE` and `FALSE` is not as simple as mapping to `MOVE`, as the right side of this map is a function, so we'd need to create internal functions.

Reply:
`TRUE`, `FALSE`, and `OR` are invalid Quil, so they can be removed.